apps/src/util/currentLocale.ts

```diff
@@ -1,5 +1,5 @@
-import {get} from 'js-cookie';
+import Localization from '@cdo/apps/localization';
 
-import {DefaultLocale} from '@cdo/generated-scripts/sharedConstants';
-
-export default () => get('language_') || DefaultLocale;
+export default () => {
+  return Localization.locale;
+};
```

Comment:
Our Localization library will now fall back to this behavior. Eventually we will replace all `currentLocale` usages to use the `Localization` class.